test/playwright/test_ux.py

```diff
@@ -147,16 +206,19 @@ def _existing_environment_interactions(
 
     # edit existing environment throught the YAML editor
     env_link.click()
+    if screenshot:
+        page.keyboard.press("PageUp")  # ensure we are at the top of the page
```

Comment:
Should this be "Home" instead of "PageUp"?

Reply:
?

Reply:
yep, done.